aws-greengrass-testing-features/aws-greengrass-testing-features-api/src/main/java/com/aws/greengrass/testing/features/DeploymentSteps.java

```diff
@@ -166,6 +237,11 @@ private void createLocalDeployment(List<List<String>> componentNames, int retryC
             commandArgs.add(entry.getKey() + "=" + entry.getValue().componentVersion());
         }
 
+        executeCommand(retryCount, commandArgs);
+    }
+
+    private void executeCommand(int retryCount, List<String> commandArgs)
+            throws InterruptedException {
         try {
             String response = platform.commands().executeToString(CommandInput.builder()
```

Comment:
This won't work for unix and mac if the configuration contains json strings like [this](https://github.com/aws-greengrass/aws-greengrass-shadow-manager/blob/main/uat/testing-features/src/main/resources/greengrass/features/shadow-1.feature#L24) as we format the command to replace `\` with `/` when we use the existing command to execute ([link to the code](https://github.com/aws-greengrass/aws-greengrass-testing/blob/04e875afa97ed80f556189ac9bc3e053c3957eb1/aws-greengrass-testing-platform/aws-greengrass-testing-platform-api/src/main/java/com/aws/greengrass/testing/platform/UnixCommands.java#L64)).